packages/renderer/src/lib/pod/DeployPodToKube.svelte

```diff
@@ -40,9 +41,14 @@ onMount(async () => {
   // grab current namespace
   currentNamespace = await window.kubernetesGetCurrentNamespace();
 
+  // check that the variable is set to a value, otherwise set to default namespace
+  if (!currentNamespace) {
+    currentNamespace = 'default';
```

Comment:
Yeah, you're right. As the function:

```
 getCurrentNamespace(): string | undefined {
    return this.currrentNamespace;
  }
 ```
 
 Would return undefined. This LGTM. From my testing it works. 
 
 Maybe put `default` as a const at the top of the file?

Reply:
Forgive me, I dont know JS at all, but wouldnt setting it as a const, prevent the selector from binding a new value to it?